### ISSUE TITLE
AlertCircle Icon Import Example error

### ISSUE THREAD

**Issue by tayambamwanza** [avmaisak/ngx-bootstrap-icons#47]
When I try to import AlertCircle as shown in the readme:

```ts
import { Alarm, AlarmFill, AlertCircle } from 'ngx-bootstrap-icons';
// Select some icons (use an object, not an array)
const icons = {
  Alarm,
  AlarmFill,
  AlertCircle
};
``` 

I get an error message

``` 
All imports in import declaration are unused.ts(6192)
'AlertCircle' is defined but never used.eslint@typescript-eslint/no-unused-vars
Module '"../../../../../../../../node_modules/ngx-bootstrap-icons/ngx-bootstrap-icons"' has no exported member
```

The others work.

**Comment by avmaisak** [comment]
@tayambamwanza this is a readme issue. AlertCircle icon does not exist. I fixed Readme file.

PLease, see all available icons **[here](https://avmaisak.github.io/ngx-bootstrap-icons/)** and i **[official site](https://icons.getbootstrap.com/)**

**Comment by avmaisak** [comment]
@tayambamwanza will fixed in 1.2.2 release
